geode-core/src/distributedTest/java/org/apache/geode/internal/BackwardCompatibilitySerializationDUnitTest.java

```diff
@@ -59,11 +56,11 @@
   private transient ByteArrayInputStream bais;
 
   public static boolean toDataCalled = false;
-  public static boolean toDataPre66Called = false;
-  public static boolean toDataPre70called = false;
+  public static boolean toDataPre11Called = false;
+  public static boolean toDataPre15called = false;
   public static boolean fromDataCalled = false;
-  public static boolean fromDataPre66Called = false;
-  public static boolean fromDataPre70Called = false;
+  public static boolean fromDataPre11Called = false;
+  public static boolean fromDataPre15Called = false;
```

Comment:
I see what was `66` is now `11` and what was `70` is now `15`. What is the significance of that mapping? How did you pick those replacements?

I get that you didn't want this test testing deprecated versions any more. But why test `1_1_0`and `1_5_0` and not say `1_13_0` and `1_14_0`? Is the choice arbitrary?

Reply:
Yup, totally arbitrary really.

Reply:
ok.